api/machine/machine.proto

```diff
@@ -713,3 +715,23 @@ message EtcdMemberList {
   repeated string members = 2;
 }
 message EtcdMemberListResponse { repeated EtcdMemberList messages = 1; }
+
+// rpc generateConfiguration
+// GenerateConfigurationRequest describes a request to generate a new configuration
+// on a node.
+message GenerateConfigurationRequest {
```

Comment:
I think we need configuration version here as the first argument to support new config formats later

Reply:
oh `version` is config version, but it's not obvious

Reply:
will rename it to config_version